scripts/hf_sim.py

```diff
@@ -528,16 +543,18 @@ def validate_end(idx_n):
     t0 = MPI.Wtime()
     in_stats = mkstemp()[1]
 
-    vm = args.hf_vel_mod_1d
+    v1d_path = args.hf_vel_mod_1d
     for s in range(work.size):
-        if args.site_vm_dir != None:
-            vm = os.path.join(args.site_vm_dir, "%s.1d" % (stations_todo[s]["name"]))
+        if args.site_v1d_dir != None:
+            v1d_path = os.path.join(
+                args.site_v1d_dir, "%s.1d" % (stations_todo[s]["name"].decode("ascii"))
```

Comment:
Should use an f string. Or at least .format

Reply:
Done